lib/woot_message_seeder.rb

```diff
@@ -76,11 +76,10 @@ def self.create_sample_form_message(conversation)
       content: 'form',
       content_attributes: {
         "items": [
-          { "name": 'email', "placeholder": 'Please enter your email', "type": 'email', "label": 'Email' },
-          { "name": 'text_area', "placeholder": 'Please enter text', "type": 'text_area', "label": 'Large Text' },
-          { "name": 'text', "placeholder": 'Please enter text', "type": 'text', "label": 'text', "default": 'defaut value' },
-          { "name": 'select', "label": 'Select Option', "type": 'select', "options": [{ "label": '🌯 Burito', "value": 'Burito' },
```

Comment:
I think the select example got removed. let's add this back

Reply:
Yeah sure.